demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1993,12 +1955,9 @@ export class DemoMeetingApp
       ) {
         this.contentShareStop();
       }
-      if (!this.roster[attendeeId] || !this.roster[attendeeId].name) {
-        this.roster[attendeeId] = {
-          ...this.roster[attendeeId],
-          ... { name: externalUserId.split('#').slice(-1)[0] + (isContentAttendee ? ' «Content»' : '') }
-        };
-      }
+      const attendeeName =  externalUserId.split('#').slice(-1)[0] + (isContentAttendee ? ' «Content»' : '');
```

Comment:
This line should move in as part of the roster component

Reply:
The reason, I did not move it in roster component as I wanted to avoid passing the logic related to `isContentAttendee` in the Roster component. As it is not necessarily related to that roster attendee.